File: Insertion Sort/insertion_sort.py
def InsertionSort(arr):
    for i in range(1, len(arr)):
        for j in range(i, 0, -1):
            if(arr[j-1] <= arr[j]):
                break
            arr[j], arr[j-1] = arr[j-1], arr[j]
    return arr

File: Insertion Sort/insertion_sort.cpp
void InsertionSort(vector<int>& arr) {
    for (int i = 1; i < arr.size(); ++i)
        for (int j = i; j >= 1; --j)
        {
            if (arr[j - 1] <= arr[j])
                break;
            swap(arr[j], arr[j - 1]);
        }
}